sdk/identity/Azure.Identity/tests/HttpPipelineMessageTest.cs

```diff
@@ -12,11 +12,13 @@ namespace Azure.Core.Tests
 {
     public class HttpPipelineMessageTest
     {
+        private static ResponseClassifier _classifier = new ResponseClassifier();
```

Comment:
I'm curious why this change is needed

Reply:
It was using ResponseClassifier.Shared before, which looked like it just created a new static instance.  For a test, it felt like creating a copy in the test would be ok, but let me know if you disagree.

Reply:
no real concerns, though if they are functionally equivalent, I assume that the static one allocates fewer times if used in other places.

Reply:
I agree with your assessment.  For now, I'm not sure it's worth keeping the file in shared source just to reduce allocations in a test, though.  If we think it's an issue later, we can add that to the list of things we want to consider putting back.